doc/tasks/phplint.md

```diff
@@ -35,6 +36,12 @@ The number of jobs you wish to use for parallel processing. If no number
 is given, it is left up to parallel-lint itself, which currently
 defaults to 10.
 
+**short_open_tag**
+
+*Default: false*
+
+This option will change short_open_tag config. 
```

Comment:
Could you please describe a bit more what is doing this option you're trying to introduce ?